app/views/CreditAndRefunds.scala.html

```diff
@@ -66,33 +67,25 @@
 }
 
 @creditAndRefundRefundProgressHtml(amount: BigDecimal) = {
-    @if(messages.lang.language.toLowerCase().equals("cy")) {
         <li>
             <p class="govuk-heading-s">
                 <span class="govuk-body">@messages("credit-and-refund.refundProgress-prt-1")</span>
                 @amount.toCurrencyString
                 <span class="govuk-body">@messages("credit-and-refund.refundProgress-prt-2")</span>
             </p>
         </li>
-    } else {
-        <li>
-            <p class="govuk-heading-s">@amount.toCurrencyString
-                <span class="govuk-body">@messages("credit-and-refund.refundProgress")</span>
-            </p>
-        </li>
-    }
 }
 
 @maybeCreditChargesHead = @{
     creditCharges.headOption.getOrElse(throw MissingFieldException("Credit Charges"))
 }
 
 @maybeBalancesDetails = @{
-    balance.getOrElse(throw MissingFieldException("Balance"))
+balance.getOrElse(throw MissingFieldException("Balance"))
 }
 
 @maybeAvailableCredit = @{
-    maybeBalancesDetails.availableCredit.getOrElse(throw MissingFieldException("Available Credit"))
+    maybeBalancesDetails.availableCredit.getOrElse(BigDecimal(0))
```

Comment:
assuming this is a part of requirement, just checking

Reply:
Yes, we want to trat None as 0